gooddata-sdk/gooddata_sdk/catalog_model/workspace.py

```diff
@@ -0,0 +1,13 @@
+# (C) 2022 GoodData Corporation
+from __future__ import annotations
+
+from gooddata_sdk.catalog_model.entity import CatalogEntity
+
+
+class CatalogWorkspace(CatalogEntity):
```

Comment:
Could it be named just `Workspace`?

Reply:
All objects have prefix Catalog. There are also objects called Attribute, ..., which are used for different use cases.
I wanted to keep the status quo.
I suggest to do another round of refactor later.

Reply:
We agreed prefixing these objects with Catalog and keep compute objects without prefix.